### PR TITLE
Some more small corrections for the GoDaddy guide.

### PR DESCRIPTION
**Pull request by therealmrmuffin**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 site_name: SimpleLogin Docs
-site_url: https://simplelogin.io/docs/
+site_url: https://simplelogin.io/
```
Comment (nguyenkims) [inline mkdocs.yml:2]: I think we need to revert the change in this file :)

Comment (therealmrmuffin) [reply mkdocs.yml:2]: > I think we need to revert the change in this file :)

Yeah, makes sense after you told me what it does.

**Review by nguyenkims — APPROVED** [review]
